UHDD/src/application/DBConnector.java

```diff
@@ -207,20 +210,102 @@ public void executeQuery(String sql) throws SQLException {
 			statement.executeQuery(sql);
 	    }
 		
-		public void createUserExecuteQuery(String username, String passwordHash, String params, String email, String role, String dateString)
-				  throws SQLException {
-					String sql = "INSERT INTO user_details (username, password_hash, password_params, email, role, user_creation_date, password_last_modified) VALUES (?, ?, ?, ?, ?, ?, ?)";
-					PreparedStatement statement = connection.prepareStatement(sql);
-					statement.setString(1, username);
-					statement.setString(2, passwordHash);
-					statement.setString(3, params);
-					statement.setString(4, email);
-					statement.setString(5, role);
-					statement.setString(6, dateString);
-					statement.setString(7, dateString);
-					statement.executeUpdate();
-					
-				}
+		public void createUserExecuteQuery(String username, String passwordHash, String params, String email, String role, String dateString, String OTPSecretKey)
+		  throws SQLException {
+			String sql = "INSERT INTO user_details (username, password_hash, password_params, email, role, user_creation_date, password_last_modified, OTPSecretKey) VALUES (?, ?, ?, ?, ?, ?, ?, ?)";
+			PreparedStatement statement = connection.prepareStatement(sql);

```

Comment:
I failed to create a new user.
Error: "email failed to send"
Some one else needs to test the new user creation